### ISSUE TITLE
Is it possible to run BINN on regression problems?

### ISSUE THREAD

**Issue by jasonleongbio** [InfectionMedicineProteomics/BINN#56]
Dear authors of the BINN paper,

Thank you for developing this package to implement biologically interpretable neural networks! 
We are so excited to try it out with our own data, and are very grateful for the user-friendly tutorial notebook accompanied by the package. 

As we were exploring the tutorial and the example dataset, we saw that the required "design matrix" is expected to include a "group" design, which suggests that by default, it deals with a classification problem. 
However, our problem is a regression problem where we want to predict some scores from the input genetic data. 

I think this has been briefly mentioned in #45 , but I'm not sure if any additional settings need to be changed if the goal is to run a regression problem. 
May I know if we can get any advice from your team regarding this?

(I actually tried running the P-NET code in Elmarakeby et al. (2021) before with our data some time ago, and remember that to run their code, a lot of settings need to be changed when shifting from the default classification mode to the regression mode. My intuition is that the BINN implementation is much clearer and straightforward, which can avoid mis-interpretation of the results.)

Thank you very much in advance ^o^

Best regards,
Jason Leong

**Comment by ErikHartman** [comment]
Hi! Sorry for the very delayed response. 

The BINN package does currently NOT support regression, although it would be straight-forward to implement. The "group" column in the design matrix would just be the real vale your regressing, and the loss should be changed from cross entropy to something like MSE. Otherwise, everything should be the same!
